### ISSUE TITLE
Intellisense not working

### ISSUE THREAD

**Issue by MyronLi** [microsoft/vscode#87680]
Issue Type: <b>Bug</b>

If the workspace contains a project creating with create-react-app,  intellisense is not working(always "loading...."). 

VS Code version: Code 1.41.0 (9579eda04fdb3a9bba2750f15193e5fafe16b959, 2019-12-11T18:37:42.077Z)
OS version: Windows_NT x64 6.1.7601

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-2450M CPU @ 2.50GHz (4 x 2494)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: unavailable_off<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: unavailable_off<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: unavailable_off|
|Load (avg)|undefined|
|Memory (System)|7.94GB (1.98GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (7)</summary>

Extension|Author (truncated)|Version
---|---|---
npm-intellisense|chr|1.3.0
vscode-eslint|dba|2.0.11
vscode-html-css|ecm|0.2.3
vscode-npm-script|eg2|0.3.10
prettier-vscode|esb|3.18.0
vscode-typescript-tslint-plugin|ms-|1.2.3
debugger-for-chrome|msj|4.12.3

</details>

**Comment by inifawaz** [comment]
I also experienced the same thing

**Comment by sonivipul89** [comment]
same thing here

**Comment by fushihara** [comment]
I am in a similar state.
Even if you press ctrl + space, only loading is displayed, and it takes about 10 seconds until intellisence is displayed.
Also, the update of the underline meaning of the error warning in the source code is very painful, about 10 seconds later.
I've never had to wait on the same machine, on the same project.
typescript 3.7.3

**Comment by asas55858995** [comment]
There is no intellisense in the React file, other files are normal. The vue or angular files have not been tried!

**Comment by sonivipul89** [comment]
for me it works after installing npm and npm intellisense extensions n my new VSCode..
try this if you are working on react-native ,react , or node.js

**Comment by MyronLi** [comment]
> for me it works after installing npm and npm intellisense extensions in my new VSCode..
> try this if you are working on react-native ,react , or node.js

try react that creating by create-react-app.

**Comment by asas55858995** [comment]
@sonivipul89 Back to version 1.40 intellisense is work. This did not take effect in the latest version 1.41.
I had to disable automatic updates and keep them at version 1.40. Wait until officials fix the problem and then update to the latest version.

**Comment by asas55858995** [comment]
@MyronLi It's a project that's been generated with React scaffolding, But it doesn't have much to do with how the project is generated. 
There should be a problem with the internal build of vscode, Look forward to the official restoration of it. 

> Note: In version 1.40, this problem did not occur

**Comment by mjbvz** [comment]
@MyronLi Is your core stored on something other than the `c:` drive?

**Comment by MyronLi** [comment]
Everything for vscode store on c: drive. @mjbvz
Installation dir: installation "C:\Program Files\Microsoft VS Code\Code.exe"
Config dir: C:\Users\%USERNAME%\.vscode
Data dir: C:\Users\%USERNAME%\AppData\Roaming\Code
But npm install on d: drive
package.json
{
  "name": "react-app-demo",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.4.0",
    "@testing-library/user-event": "^7.2.0",
    "classnames": "^2.2.6",
    "react": "^16.12.0",
    "react-dom": "^16.12.0",
    "react-scripts": "3.3.0"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}

**Comment by MyronLi** [comment]
node version: v10.17.0    manged by nvm-windows. @mjbvz

**Comment by VitaminCtea** [comment]
There are two solutions to the problem of intellisense not working:

1. Back to version 1.40, and then disable automatic updates in the settings.json file.
2. If you're friends with version 1.41, you can fix the problem by installing the [JavaScript and TypeScript Nightly](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-next) plugin

> **Self test, both are effective.** I hope I can help others who have the same problem as me

**Comment by MyronLi** [comment]
Thank you for feedback. @VitaminCtea 
I had backed to version 1.40.

**Comment by zhanyuzhang** [comment]
@VitaminCtea  It's now normal After installing [JavaScript and TypeScript Nightly](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-next) .Ths

**Comment by VitaminCtea** [comment]
@MyronLi @zhanyuzhang Well done!

**Comment by mjbvz** [comment]
Duplicate of https://github.com/microsoft/TypeScript/issues/35512

Should be fixed in VS Code insiders
